### PR TITLE
docs(examples): update Tailwind CSS example

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Bringing the Tailwind CSS example in line with the new templates + changes I made in #2143

### CODE REVIEW

File: examples/tailwindcss/package.json [hunk 1]
```diff
@@ -1,35 +1,38 @@
...
+    "dev:remix": "cross-env NODE_ENV=development remix dev",
+    "generate:css": "npx tailwindcss -o ./app/tailwind.css",
     "postinstall": "remix setup node",
-    "start": "remix-serve build"
+    "start": "cross-env NODE_ENV=production remix-serve build"
   },
   "dependencies": {
     "@remix-run/react": "1.2.2",
     "@remix-run/serve": "1.2.2",
+    "cross-env": "^7.0.3",
```
Comment (Hamatek) [inline examples/tailwindcss/package.json:21]: maybe we could move it to devDependencies

Comment (MichaelDeBoey) [reply examples/tailwindcss/package.json:21]: Just copied over the base template, but should indeed be in `devDependencies` 👍

Fixed the template in #2157

Comment (jacob-ebey) [reply examples/tailwindcss/package.json]: `cross-env` is used in the start script, it is in fact a runtime dependency and therefore belongs in `dependencies` not `devDependencies`.

Comment (MichaelDeBoey) [reply examples/tailwindcss/package.json]: @jacob-ebey Shouldn't it be a dependency of `remix` itself then? 🤔

**Pull request merged**